docs/source/07_extend_kedro/03_custom_datasets.md

```diff
@@ -255,16 +256,14 @@ class ImageDataSet(AbstractDataSet):
         Returns:
             Data from the image file as a numpy array
         """
-        # using get_filepath_str ensures that the protocol and path are appended correctly for different filesystems
```

Comment:
Why remove these comments?

Reply:
To clear up the diff compared to the versioned example (which is now _sort of_ automatically generated). They are already given as comments in the individual examples and so don't don't really add anything when showing the complete example.